server/middleware/barcode/barcodeAuthorisationMiddleware.ts

```diff
@@ -0,0 +1,32 @@
+import { RequestHandler } from 'express'
+import type { VerifyErrors, JwtPayload } from 'jsonwebtoken'
+import { verify } from 'jsonwebtoken'
+import config from '../../config'
+import logger from '../../../logger'
+
+export default function barcodeAuthorisationMiddleware(): RequestHandler {
+  return (req, res, next) => {
+    res.locals.barcodeUser = undefined
+
+    if (!req.session.createBarcodeAuthToken) {
+      logger.warn('Attempt to access protected barcode resource without an auth token')
```

Comment:
If the users bookmark the barcode resource then we could see this warning a lot - so I'm not sure it needs to be logged, BAU?

Reply:
Thats a fair shout - I'll remove that